_articles/manage-repos-ubuntu.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: article
+title: Manage Repositories in Ubuntu
+description: >
+  Add, remove and edit repositories in Ubuntu
+keywords:
+  - Ubuntu 18.04
+  - Ubuntu 19.10
+  - System76
+  - apt
+image: http://support.system76.com/images/pop-icon.png
+hidden: false
+section: pop-ubuntu
```

Comment:
ubuntu not pop-ubuntu

Reply:
Fixed here: https://github.com/system76/docs/pull/334